examples/fuel-explorer/fuel-explorer/schema/fuel_explorer.schema.graphql

```diff
@@ -43,17 +43,20 @@ type Genesis {
   coins_root: Bytes32!
   contracts_root: Bytes32!
   messages_root: Bytes32!
+  is_genesis: Boolean!
```

Comment:
I may be missing something, but what is the purpose of these boolean flags? Is there something in the `union` parsing that relies upon them?

Reply:
So that you can query a union for the single type that you want. (I also included the label field, which you could also use, but more simple to just use the boolean)